apps/console/src/auth.html

```diff
@@ -558,7 +558,13 @@
 
         <!-- Start of custom scripts added to the head -->
 
-        <script defer type="text/javascript" src="<%= htmlWebpackPlugin.options.publicPath %>extensions/head-script.js"></script>
+        <script
+            id="head-script"
```

Comment:
Lets keep the `defer` at the top.

Reply:
Updated via: 26f2cf37e27f61532fc661befa2b7b35c734fd70